torch/csrc/jit/codegen/cuda/type.cpp

```diff
@@ -689,6 +689,10 @@ static const char* rng_op_type_inline_op2string(RNGOpType t) {
       return "rng_uniform";
     case RNGOpType::UniformRange:
       return "rng_uniform_range";
+    case RNGOpType::Normal:
+      return "rng_normal";
+    case RNGOpType::NormalRange:
```

Comment:
I'm not sure if this is a good name? `Normal` is standard normal distribution with mean 0 and std 1. `NormalRange` is with custom mean and std.

Reply:
I think better names are "standard normal" and "general normal"? https://en.wikipedia.org/wiki/Normal_distribution#Definitions